core/editor/src/document/document_message_handler.rs

```diff
@@ -142,15 +142,15 @@ impl DocumentMessageHandler {
 	}
 
 	/// Returns the paths to all layers in order, optionally including only selected layers
-	fn layers_sorted(&self, only_selected: bool) -> Vec<Vec<LayerId>> {
+	fn layers_sorted(&self, selected: Option<bool>) -> Vec<Vec<LayerId>> {
```

Comment:
Is `Option<bool>` the conventional method of having an optional bool arg? You are essentially requesting two copies of the same information, and the case of `Some(false)` doesn't really make sense right? Would `Option<()>` work? Perhaps this is a non-issue, it just strikes as a little unclean.

Reply:
Maybe I'm wrong though, feel free to ignore.

Reply:
I basically want to differentiate three states None => All layers, Some(true) => selectedl ayers, Some(false) => non_selected layers